hris_publisher/dev-vars.yml

```diff
@@ -6,7 +6,7 @@ CIS_IAM_ROLE_ARN: arn:aws:iam::656532927350:role/CISPublisherRole
 CIS_PUBLISHER_NAME: hris
 CIS_IAM_ROLE_SESSION_NAME: dev_hris_client
 CIS_LAMBDA_VALIDATOR_ARN: arn:aws:lambda:us-west-2:656532927350:function:cis_functions_stage_validator
-CIS_PERSON_API_URL: https://person-api.sso.allizom.org
+CIS_PERSON_API_URL: person-api.sso.allizom.org
```

Comment:
i dont understand why the URL looks like an audience and the audience like an URL :)
the later is technically valid, but the former is specially odd